examples/LEVDisplay/LEVDisplay.ino

```diff
@@ -0,0 +1,207 @@
+/**********************************************
+ * AntPlus LEV Display example
+ *
+ * Finds a nearby LEV Sensor, pairs
+ * to it and then reads the information
+ * out via the serial port.
+ *
+ * Author Bernd Wok�ck
+ *        based on the work of Curtis Malainey
+ **********************************************/
+#include <Arduino.h>
+#include "ANT.h"
+#include "ANTPLUS.h"
+
+#define BAUD_RATE 9600
+#define CHANNEL_0 0
+#define CHANNEL_1 1
```

Comment:
this isnt used anywhere, best to remove it so we don't confuse anyone

Reply:
Removed